.ci/appveyor.yml

```diff
@@ -39,6 +39,7 @@ install:
   # Check that we have the expected version and architecture for Python
   - "python --version"
   - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
+  - "pear install PHP_CodeSniffer"
```

Comment:
add the skip decorator to your test class, so you dont need to install pear onto windows.